magento/graphql-ce
# Render category description directives

### Description
This PR improves output for description of a category. Previously we just had a raw output. Now, if you use some directive (upon uploading an image), its contents are rendered

### Fixed Issues (if relevant)

1. https://github.com/magento/graphql-ce/issues/135: GraphQL API returns rendered content for category

### Manual testing scenarios
1. Magento installation from 2.3-develop branch (optionally: with sample data installed)
2. Open a category and upload some image in the category description
3. Make a GraphQL request with category description included:

```
{
  category(id: 15) {
    name
    description
  }
}
```
4. In the description, you should see full image URL instead of `{media url=...}`

Please note, not all directives are supported in the category description on the core level. For example, you cannot insert a block like in https://github.com/magento/graphql-ce/pull/140 . It's not the current implementation limitation but core behavior. If you try to insert a block and check the storefront, it's not rendered there as well. All directives that are supported by default template filter (used on storefront) are also supported in the GraphQL response.
